File: shared/src/main/java/xyz/klinker/messenger/shared/view/MaterialPreferenceCategory.java
package xyz.klinker.messenger.shared.view;

import android.content.Context;
import android.preference.PreferenceCategory;
import android.util.AttributeSet;

import xyz.klinker.messenger.shared.R;

public class MaterialPreferenceCategory extends PreferenceCategory {

    public MaterialPreferenceCategory(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        init();
    }

    public MaterialPreferenceCategory(Context context, AttributeSet attrs) {
        super(context, attrs);
        init();
    }

    public MaterialPreferenceCategory(Context context) {
        super(context);
        init();
    }

    private void init() {
        setLayoutResource(R.layout.preference_category_card);
    }
}

File: shared/src/main/java/xyz/klinker/messenger/shared/view/MaterialPreferenceCategory.kt
package xyz.klinker.messenger.shared.view

import android.content.Context
import android.preference.PreferenceCategory
import android.support.v14.preference.SwitchPreference
import android.util.AttributeSet

import xyz.klinker.messenger.shared.R

class MaterialPreferenceCategory : PreferenceCategory {

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr)
    constructor(context: Context, attrs: AttributeSet) : super(context, attrs)
    constructor(context: Context) : super(context)


    init {
        layoutResource = R.layout.preference_category_card
    }
}
